src/patternfly/components/Alert/examples/danger-alert.hbs

```diff
@@ -1 +1,34 @@
-{{#> alert modifierClass="pf-is-danger"}}<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>{{/alert}}
\ No newline at end of file
+{{#> alert pf-c-alert--modifierClass="pf-m-danger pf-m-top-aligned"}}
+ <div class="pf-c-alert__icon">
+   <i class="fas fa-exclamation-circle"></i>
+ </div>
+  <div class="pf-c-alert__body">
+    <h4 class="pf-c-alert__title">Danger notification title</h4>
+    <p class="pf-c-alert__description">This is a description of the notification content.</p>
+  </div>
+{{/alert}}
+<br/>
```

Comment:
I don't really like this but if you really need to add it please use `<br>`.

HTML doesn't have auto close tags anymore.

Reply:
👍